sympy/functions/elementary/tests/test_piecewise.py

```diff
@@ -622,6 +622,7 @@ def test_piecewise_interval():
         (S.Half, True))
 
 
+@slow
 def test_piecewise_collapse():
```

Comment:
Has this become slower or are you just adding the flag because it was already slow?

Reply:
On my slow system it was very slow. I'll try remove it and see if it gets flagged by the test suite.

Reply:
For me this test takes 0.75 seconds on both master and the PR (with or without gmpy2 being installed):
```console
$ pytest sympy/functions/elementary/tests/test_piecewise.py::test_piecewise_collapse
======================================== test session starts ========================================
platform linux -- Python 3.7.10, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
architecture: 64-bit
cache:        yes
ground types: python 

rootdir: /home/oscar/current/sympy/sympy.git, configfile: pytest.ini
plugins: xdist-2.3.0, forked-1.3.0
collected 1 item                                                                                    

sympy/functions/elementary/tests/test_piecewise.py .                                          [100%]

========================================= 1 passed in 0.75s =========================================
```

Reply:
all slows reverted